cpp/include/copc-lib/hierarchy/key.hpp

```diff
@@ -24,8 +25,13 @@ class Box;
 class VoxelKey
 {
   public:
-    VoxelKey(int32_t d, int32_t x, int32_t y, int32_t z) : d(d), x(x), y(y), z(z) {}
+    VoxelKey(const int32_t &d, const int32_t &x, const int32_t &y, const int32_t &z) : d(d), x(x), y(y), z(z) {}
```

Comment:
andrew is right, int32_t is smaller than a pointer to an int32_t object 😆 

but it doesn't really matter